### PR TITLE
Remove imported assets during tests and fix Lookup tests

### PR DESCRIPTION
**Pull request by CalebBredekamp**
The Import tests were the issue for `AssetControllerQueryAPITest.queryByTags` since the import dataset now includes tags. So I've just added a section to delete the imported assets.

The Lookup tests on the other hand were expecting a description which is null. But it looks like if a description is null in the DB, its ignored during the return, hence the tests failing. Assuming this is desired behaviour I've just removed the expectation from the test where the description is null.

### CODE REVIEW

File: Service/src/test/java/za/co/imqs/api/TestImport.java [hunk 1]
```diff
@@ -28,7 +29,35 @@
     @After
     @Override
     public void after() {
-        // Override base class to do nothing
+        // Override base class to delete the assets imported
+        UUID[] assets = {
```
Comment (frankvanrietimqs) [inline Service/src/test/java/za/co/imqs/api/TestImport.java:33]: This is terribly fragile as it depends on the contents of the file. I would suggest that you make use of the AssetTestingController delete endpoint to remove uuids that you have gotten by querying the database for the available uuids or perhaps cleaner add an endpoint that will do a blanket delete

**Review by frankvanrietimqs — CHANGES_REQUESTED** [review]
Please see comment

File: Service/src/main/java/za/co/imqs/coreservice/controller/AssetTestingController.java [hunk 2]
```diff
@@ -49,4 +57,15 @@ public ResponseEntity delete(@PathVariable String uuid) {
         assetWriter.obliterateAssets(asUUID(uuid));
         return new ResponseEntity(HttpStatus.OK);
     }
+
+    @RequestMapping(
+            method = RequestMethod.DELETE, value = "/clear-db"
+    )
+    public ResponseEntity blanketDelete() {
+        final List<UUID> uuids =
```
Comment (frankvanrietimqs) [inline Service/src/main/java/za/co/imqs/coreservice/controller/AssetTestingController.java:65]: This is test code so everything does not have to bwe super correct, but lets use this as a learning opportunity.

I suggest you move getting the list of uuids into the repository layer. The layering s all about encapsulation and abstraction.  By having the query here knowledge about what the database internals look like are escaping the repository layer.

Comment (CalebBredekamp) [reply Service/src/main/java/za/co/imqs/coreservice/controller/AssetTestingController.java:65]: Alrighty, will do

File: Service/src/main/java/za/co/imqs/coreservice/controller/AssetTestingController.java [hunk 3]
```diff
@@ -62,9 +61,7 @@ public ResponseEntity delete(@PathVariable String uuid) {
             method = RequestMethod.DELETE, value = "/clear-db"
     )
     public ResponseEntity blanketDelete() {
-        final List<UUID> uuids =
-                new JdbcTemplate(this.assetDs).queryForList("SELECT asset_id FROM dtpw.dtpw_export_view;",
-                        UUID.class);
+        final List<UUID> uuids = assetReader.getAllAssets();
```
Comment (frankvanrietimqs) [inline Service/src/main/java/za/co/imqs/coreservice/controller/AssetTestingController.java:64]: Closer :) 
Consider that you are not being picky on what you are deleting. Would it not be much simpler to add a deleteAll method to CoreAssetWriter?

**Review by frankvanrietimqs — APPROVED** [review]

**Pull request merged**